.github/workflows/ci.yaml

```
@@ -39,25 +38,55 @@ jobs:
       - name: Clone OpenAssetIO
         uses: actions/checkout@v2
         with:
-          repository: TheFoundryVisionmongers/OpenAssetIO
+          repository: foundrytom/OpenAssetIO
+          ref: work/347-pythonVenvBuild
           path: dependencies/OpenAssetIO
 
+      - name: Cache conan packages
+        id: cache-conan
+        uses: actions/cache@v2
+        with:
+          path: ~/conan
```

[fix] It's a bit confusing, but `conan install --install-folder ~/conan ...` only tells conan where to put generated config files (e.g. CMake scripts).  By default the actual packages are installed in `~/.conan` (note the `.`).  

In the Vagrant bootstrap script(s) in OpenAssetIO there is a line `export CONAN_USER_HOME="$HOME/conan"` - that's the bit that tells conan where to install packages to.  In OpenAssetIO the packages and generated scripts are put in the same location, simply for convenience.

So we have a few options. The simplest is to just change this line to 
```suggestion
          path: ~/.conan
```
It means the generated files are not cached, but then they're probably regenerated every time anyway.

Or we could set the `CONAN_USER_HOME` environment variable, like in OpenAssetIO, to be explicit.

---

Ahhhhhh, awesome thanks, I did wonder if that var was a DRY device, or actually also doing Conan things. I like your suggestion, given hopefully all this will go soon.